src/Components/test/testassets/Components.TestServer/RazorComponentEndpointsStartup.cs

```diff
@@ -22,7 +22,12 @@ public RazorComponentEndpointsStartup(IConfiguration configuration)
     // This method gets called by the runtime. Use this method to add services to the container.
     public void ConfigureServices(IServiceCollection services)
     {
-        services.AddRazorComponents();
+        services.AddRazorComponents()
+            .AddServerComponents()
+            .AddWebAssemblyComponents(options =>
+            {
+                options.FrameworkFilesPathPrefix = "/WasmMinimal";
+            });
```

Comment:
This is so that we don't inherit the baggage from `BasicTestApp` when all we need is a super minimal WebAssembly app to enable WebAssembly interactivity.